src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -52,8 +54,10 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        // Don't allow adding of lessons with the adding of person.
+        Set<Lesson> lessonSet = new HashSet<>();
```

Comment:
can you rename to be consistent with `tagList`
```suggestion
        Set<Lesson> lessonList = new HashSet<>();
```

Reply:
@eeliana by the way, sets are unordered right. Shouldn't we use an ordered data structure instead?

Reply:
maybe need to use `TreeSet`. It can auto sort for us, just need to make `Lesson` a `Comparable`.

Reply:
so we just using `Set` and sort later is it?

Reply:
I think we can go with TreeSet for now by others' inputs as well

Reply:
@eeliana seems like TreeSet doesn't use equals method of Lesson but considers Lesson to be equal if compareTo == 0